control-plane/subcommand/gateway-resources/command_test.go

```diff
@@ -254,3 +268,130 @@ func TestRun(t *testing.T) {
 		})
 	}
 }
+
+var validGWConfiguration = `gatewayClassConfigs:
+  - apiVersion: mesh.consul.hashicorp.com/v2beta1
+    metadata:
+      name: consul-mesh-gateway
+      namespace: namespace
+    kind: gatewayClassConfig
+    spec:
+      deployment:
+meshGateways:
+  - name: mesh-gateway
+    spec:
+      gatewayClassName: consul-mesh-gateway
+`
+
+var invalidGWConfiguration = `gatewayClassConfigs:
+  - apiVersion: mesh.consul.hashicorp.com/v2beta1
+metadata:
+      namespace: namespace
+    kind: gatewayClassConfig
+    spec:
+      deployment:
+meshGateways:
+  - name: mesh-gateway
+    spec:
+      gatewayClassName: consul-mesh-gateway
+`
+
+func TestRun_loadMesGatewayConfigs(t *testing.T) {
```

Comment:
typo: Missing the h in mesh `TestRun_loadMesGatewayConfigs` -> `TestRun_loadMeshGatewayConfigs`

Reply:
good catch, these actually shouldn't even read `Mesh` anymore